crowbar_framework/app/views/backup/index.html.haml

```diff
@@ -0,0 +1,52 @@
+.row
```

Comment:
FWIW, I don't think we will need the full UI (as there's no restore for Cloud 5). It's useful for now, but once we get the upgrade UI, I think we can drop that.